lambdamud-client/src/App.js

```diff
@@ -13,10 +13,6 @@ import MainScreen from './components/MainScreen';
 
 class App extends Component {
 
```

Comment:
Why'd you remove this? I feel like it should've worked, all  you'd have to do after that is add a window.location.reload() call after it to get it to show the login screen.